internal/pkg/propagation/propagation.go

```diff
@@ -258,6 +258,9 @@ func (prop *Propagation) visitCall(call *ssa.Call, maxInstrReached map[*ssa.Basi
 	// Source methods that return tainted values regardless of their arguments should be identified by the fieldpropagator analyzer.
 	if recv := call.Call.Signature().Recv(); recv != nil && sourcetype.IsSourceType(prop.config, prop.taggedFields, recv.Type()) {
 		visitingFromArg := false
+		if len(call.Call.Args) == 0 {
```

Comment:
Any bug fix should have a unit test? Could we add a unit test for this particular case?

Reply:
I agree. My intent here is to fix the analyzer ASAP, because currently we have a panicking analyzer on `master`. There will be a follow-up PR with a detailed investigation + tests.